bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -1561,6 +1563,42 @@ impl Client {
         let max_upload_size = self.inner.load_or_fetch_max_upload_size().await?;
         Ok(max_upload_size.into())
     }
+
+    /// Subscribe to [`RoomInfo`] updates given a provided [`RoomId`].
+    ///
+    /// This works even for rooms we haven't received yet, so we can subscribe
+    /// to this and wait until we receive updates from them when sync responses
+    /// are processed.
```

Comment:
Can you add about being careful to not maintain too many of these alive / that the tasks should be short-lived, and should be explicitly dropped as soon as they're not useful for the caller anymore?

Reply:
Done, thanks for the explanation!